pkg/receiver/rawk8seventsreceiver/receiver.go

```diff
@@ -44,13 +47,15 @@ var severityMap = map[string]plog.SeverityNumber{
 }
 
 type rawK8sEventsReceiver struct {
-	cfg              *Config
-	client           k8s.Interface
-	eventControllers []cache.Controller
-	eventCh          chan *eventChange
-	ctx              context.Context
-	cancel           context.CancelFunc
-	startTime        time.Time
+	cfg                  *Config
+	client               k8s.Interface
+	eventControllers     []cache.Controller
+	eventCh              chan *eventChange
+	ctx                  context.Context
+	cancel               context.CancelFunc
+	startTime            time.Time
+	storage              storage.Client
+	startResourceVersion uint64
```

Comment:
I think a better name for this would be `latestResourceVersion` or something similar. As is, we're writing to `startResourceVersion` for each processed record, which is unintuitive.